images/telmetry-and-insights-operator-data-flow.svg

```diff
@@ -0,0 +1 @@
3-b063-dd844e59ef04" x="69.208" y="0">.</tspan><tspan class="f5fae57e-d335-45f2-8ca4-86b74c989eb2" x="71.844" y="0">c</tspan><tspan class="b9543da0-0a47-4c94-9449-f0ad2f82f872" x="77.531" y="0">o</tspan><tspan class="b3094086-12db-4364-99d0-57eeb934f946" x="84.205" y="0">m</tspan></text><text class="bf04637f-4250-4ec4-9a01-f2796cc9edbd" transform="translate(466.001 47.788)"><tspan letter-spacing=".001em">R</tspan><tspan x="9.857" y="0" letter-spacing=".002em">e</tspan><tspan x="18.178" y="0">d </tspan><tspan class="ab813449-8bcf-44ac-a6d3-9f607c79e4bf" x="30.189" y="0">H</tspan><tspan class="fc0eeafa-17ea-4f7b-9c29-7651ca657d63" x="41.035" y="0">a</tspan><tspan x="49.026" y="0" letter-spacing="0">t</tspan></text><path class="efe767cc-f685-42ba-8267-e45e1c2fce03" d="M187 36h107v60H187z"/><text transform="translate(228.886 63.047)" letter-spacing="-.017em" font-family="RedHatText,&quot;Red Hat Text&quot;,Overpass,&quot;Helvetica Neue&quot;,Arial,sans-serif" font-weight="500" font-size="11" fill="#151515">W<tspan class="bafa8fcc-02f0-403f-a130-1bdc1dc6182d" x="10.114" y="0">e</tspan><tspan class="b3094086-12db-4364-99d0-57eeb934f946" x="16.506" y="0">b</tspan><tspan class="f5fae57e-d335-45f2-8ca4-86b74c989eb2"><tspan x="-8.432" y="12">c</tspan><tspan class="b9543da0-0a47-4c94-9449-f0ad2f82f872" x="-2.745" y="12">o</tspan><tspan class="b7afa9f9-549e-4a06-83db-fd9d2e583d35" x="3.929" y="12">n</tspan><tspan x="10.485" y="12" letter-spacing=".008em">s</tspan><tspan class="b9543da0-0a47-4c94-9449-f0ad2f82f872" x="15.838" y="12">o</tspan><tspan class="b12fccbf-5c5a-4e3f-afbe-43b42b5328ee" x="22.512" y="12">l</tspan><tspan class="b3094086-12db-4364-99d0-57eeb934f946" x="25.356" y="12">e</tspan></tspan></text><text class="e9183669-4eca-4c74-9ffa-68cfdfd29e54" transform="translate(229.89 148.047)"><tspan class="ab3be15a-311b-4b24-a630-db02a9b27cbe">I<tspan class="efd0bc69-fd43-4b0d-ba79-ccc4eec56b86" x="3.186" y="0">n</tspan><tspan class="b491f6d9-0ee4-4d0e-938a-975c64565974" x="9.741" y="0">s</tspan><tspan class="f3c1e948-5e1a-4abd-b39f-748fcbfc7dbb" x="15.129" y="0">i</tspan><tspan class="b8b0efa3-14e5-4242-9911-bf3da28696e2" x="17.967" y="0">g</tspan><tspan class="a7f58725-9640-45c3-9d82-7d43f7a2748a" x="24.781" y="0">h</tspan><tspan class="a168b056-6de6-414f-87f2-948c27c24070" x="31.149" y="0">t</tspan><tspan class="b8b02ad7-0381-4e8a-ba47-c12968a4e0f5" x="35.951" y="0">s</tspan></tspan><tspan class="b8598041-ffac-4c2a-a990-8dc864390082" x="-3.465" y="12">O</tspan><tspan class="b12fccbf-5c5a-4e3f-afbe-43b42b5328ee" x="5.614" y="12">p</tspan><tspan class="a25d49ff-26c8-4e34-8603-c6ac16c5fdb4" x="12.407" y="12">e</tspan><tspan class="a0a7f024-fcc9-4dd5-bd2a-987e80431c5c" x="18.799" y="12">r</tspan><tspan class="b4f1d1fa-8101-43ed-af93-ae6256bda5c7" x="23.17" y="12">a</tspan><tspan x="29.186" y="12" letter-spacing="-.005em">t</tspan><tspan class="b9543da0-0a47-4c94-9449-f0ad2f82f872" x="33.689" y="12">o</tspan><tspan x="40.362" y="12">r</tspan></text></svg>
```

Comment:
Should this cover a proxy (or the ability to pass through a proxy)? Not sure if a call out on this is needed or not.

Reply:
@radekvokal can you please advise? Thank you!

Reply:
@sferich888 do we have other diagrams like this one where we specifically call out proxy? From my PoV, it "just works(tm)" and I'm looking at telemetry where we also don't mention proxy. So I would reference the Cluster wide proxy page if someone needs a proxy setup and I would also mention the configuring of the firewall with correct endpoints 
https://docs.openshift.com/container-platform/4.5/installing/install_config/configuring-firewall.html#configuring-firewall

Reply:
@radekvokal no, but I also don't see any diagrams or depictions showing OpenShift working with external services. The ask here to show this working with a proxy, is mainly to help show customers, what all is possible. 

If they see that this works behind a proxy with a footnote/call out to the proxy configuration to make it work, then customer can almost intrinsically understand this works for network isolated situations (or could be turned off).